domain/src/main/java/com/teamagam/gimelgimel/domain/location/SendSelfLocationsInteractor.java

```diff
@@ -41,9 +42,10 @@ public SendSelfLocationsInteractor(ThreadExecutor threadExecutor,
   private DataSubscriptionRequest buildSendRequest(
       DataSubscriptionRequest.SubscriptionRequestFactory factory) {
     return factory.create(mLocationRepository.getLocationObservable(),
-        locationSampleObservable -> locationSampleObservable.onBackpressureLatest()
+        locationSampleObservable -> locationSampleObservable.toFlowable(BackpressureStrategy.LATEST)
             .filter(this::shouldUpdateServer)
             .map(this::createMessage)
+            .toObservable()
```

Comment:
I think that here the logical filtering is sufficient for handling a backpressure. Don't u agree?